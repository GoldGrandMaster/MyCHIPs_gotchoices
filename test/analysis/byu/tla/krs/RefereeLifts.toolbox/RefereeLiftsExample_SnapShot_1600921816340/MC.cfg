\* MV CONSTANT declarations
CONSTANTS
A = A
B = B
C = C
D = D
\* MV CONSTANT definitions
CONSTANT
Users <- const_1600921812032160000
\* CONSTANT definitions
CONSTANT
Links <- const_1600921812032161000
\* CONSTANT definitions
CONSTANT
Cycles <- const_1600921812032162000
\* CONSTANT definitions
CONSTANT
Polylines <- const_1600921812032163000
\* CONSTANT definitions
CONSTANT
PaymentsCount <- const_1600921812032164000
\* CONSTANT definitions
CONSTANT
PaymentsRange <- const_1600921812032165000
\* CONSTANT definitions
CONSTANT
LinearLiftsCount <- const_1600921812032166000
\* CONSTANT definitions
CONSTANT
NewStateLimit <- const_1600921812032167000
\* CONSTANT definitions
CONSTANT
GLOBAL_REF <- const_1600921812032168000
\* SPECIFICATION definition
SPECIFICATION
Spec
\* INVARIANT definition
INVARIANT
PartnersBalancesAreSymmetrical
ProjectedAndAvailableBalancesAreEqualWhenNoLiftsInProgress
FoilBalancesAreNotPositive
StockBalancesAreNotNegative
UsersChitsOfGoodLiftsAreSymmetrical
AvailableMoneySumIsZero
ProjectedMoneySumIsZero
\* PROPERTY definition
PROPERTY
Termination
\* Generated on Wed Sep 23 22:30:12 MDT 2020