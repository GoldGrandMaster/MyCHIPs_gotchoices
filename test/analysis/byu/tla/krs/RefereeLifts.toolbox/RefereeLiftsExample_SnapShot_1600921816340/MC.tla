---- MODULE MC ----
EXTENDS RefereeLifts, TLC

\* MV CONSTANT declarations@modelParameterConstants
CONSTANTS
A, B, C, D
----

\* MV CONSTANT definitions Users
const_1600921812032160000 == 
{A, B, C, D}
----

\* CONSTANT definitions @modelParameterConstants:0Links
const_1600921812032161000 == 
{<<A, B>>, <<B, C>>, <<C, A>>, <<B, D>>}
----

\* CONSTANT definitions @modelParameterConstants:2Cycles
const_1600921812032162000 == 
{<<A, B, C>>}
----

\* CONSTANT definitions @modelParameterConstants:3Polylines
const_1600921812032163000 == 
{<<A, B, D>>}
----

\* CONSTANT definitions @modelParameterConstants:4PaymentsCount
const_1600921812032164000 == 
8
----

\* CONSTANT definitions @modelParameterConstants:5PaymentsRange
const_1600921812032165000 == 
1..3
----

\* CONSTANT definitions @modelParameterConstants:6LinearLiftsCount
const_1600921812032166000 == 
4
----

\* CONSTANT definitions @modelParameterConstants:7NewStateLimit
const_1600921812032167000 == 
100
----

\* CONSTANT definitions @modelParameterConstants:8GLOBAL_REF
const_1600921812032168000 == 
"TRUSTED_REF_1"
----

=============================================================================
\* Modification History
\* Created Wed Sep 23 22:30:12 MDT 2020 by kylestorey
