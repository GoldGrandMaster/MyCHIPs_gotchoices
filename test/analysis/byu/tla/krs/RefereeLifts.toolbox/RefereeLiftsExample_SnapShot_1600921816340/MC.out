@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.15 of Day Month 20?? (rev: eb3ff99)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2188:0 @!@!@
Running Random Simulation with seed -2037268562999775004 with 4 workers on 4 cores with 7282MB heap and 2730MB offheap memory [pid: 4226] (Mac OS X 10.13.6 x86_64, AdoptOpenJDK 14.0.1 x86_64).
@!@!@ENDMSG 2188 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file /Users/kylestorey/workspace/MyCHIPs/test/analysis/krs/RefereeLifts.toolbox/RefereeLiftsExample/MC.tla
Parsing file /Users/kylestorey/workspace/MyCHIPs/test/analysis/krs/RefereeLifts.toolbox/RefereeLiftsExample/RefereeLifts.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/TLC.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Integers.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Sequences.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/FiniteSets.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Bags.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Naturals.tla
Semantic processing of module Naturals
Semantic processing of module Integers
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module Bags
Semantic processing of module RefereeLifts
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2020-09-23 22:30:13)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2212:0 @!@!@
Implied-temporal checking--satisfiability problem has 1 branches.
@!@!@ENDMSG 2212 @!@!@
"Init"  1
@!@!@STARTMSG 2269:0 @!@!@
Computed 1 initial states...
@!@!@ENDMSG 2269 @!@!@
"Tick"  2
"Tick"  2
@!@!@STARTMSG 2109:1 @!@!@
Successor state is not completely specified by the next-state action.

@!@!@ENDMSG 2109 @!@!@
@!@!@STARTMSG 2121:1 @!@!@
The behavior up to this point is:
@!@!@ENDMSG 2121 @!@!@
@!@!@STARTMSG 2217:4 @!@!@
1: <Initial predicate>
/\ messages = << >>
/\ lifts = (A :> << >> @@ B :> << >> @@ C :> << >> @@ D :> << >>)
/\ _stateNum = 1
/\ linearLiftsRemaining = 4
/\ paymentsRemaining = 8
/\ tallies = ( <<A, B, "Foil">> :> [chits |-> <<>>] @@
  <<A, C, "Stock">> :> [chits |-> <<>>] @@
  <<B, A, "Stock">> :> [chits |-> <<>>] @@
  <<B, C, "Foil">> :> [chits |-> <<>>] @@
  <<B, D, "Foil">> :> [chits |-> <<>>] @@
  <<C, A, "Foil">> :> [chits |-> <<>>] @@
  <<C, B, "Stock">> :> [chits |-> <<>>] @@
  <<D, B, "Stock">> :> [chits |-> <<>>] )
/\ nextGuid = 101

@!@!@ENDMSG 2217 @!@!@
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  2
"ReceivePayment"  3
"ReceivePayment"  3
"SendInvoice"  3
"SendInvoice"  3
"SendInvoice"  3
"SendInvoice"  3
"SendInvoice"  3
"SendInvoice"  3
"SendInvoice"  3
"SendInvoice"  3
"SendPayment"  4
"SendPayment"  4
"SendPayment"  4
"SendPayment"  4
"SendInvoice"  4
"SendInvoice"  4
"SendInvoice"  4
"SendInvoice"  4
"ReceivePayment"  4
"SendPayment"  5
"SendPayment"  5
"SendPayment"  5
"SendPayment"  5
"DeclineInvoice"  4
"DeclineInvoice"  4
"SendPayment"  5
"SendPayment"  5
"SendPayment"  5
"SendPayment"  5
"ReceiveDeclinedInvoice"  5
"PayInvoice"  6
"PayInvoice"  5
"SendInvoice"  7
"SendInvoice"  7
"SendInvoice"  7
"SendInvoice"  7
"Tick"  6
"Tick"  2
"Tick"  2
"Tick"  2
"Tick"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  2
"SendPayment"  6
"SendPayment"  6
"SendPayment"  6
"SendPayment"  6
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendPayment"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  2
"SendInvoice"  6
"SendInvoice"  6
"SendInvoice"  6
"SendInvoice"  6
"Tick"  2
"DeclineInvoice"  8
@!@!@STARTMSG 1000:1 @!@!@
TLC threw an unexpected exception.
This was probably caused by an error in the spec or model.
See the User Output or TLC Console for clues to what happened.
The exception was a java.lang.NullPointerException
java.lang.NullPointerException
	at tlc2.tool.TLCStateMut.fingerPrint(TLCStateMut.java:252)
	at tlc2.tool.Simulator.printBehavior(Simulator.java:394)
	at tlc2.tool.Simulator.printBehavior(Simulator.java:348)
	at tlc2.tool.Simulator.printBehavior(Simulator.java:339)
	at tlc2.tool.Simulator.simulate(Simulator.java:285)
	at tlc2.TLC.process(TLC.java:1051)
	at tlc2.TLC.main(TLC.java:289)

@!@!@ENDMSG 1000 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 2320ms at (2020-09-23 22:30:14)
@!@!@ENDMSG 2186 @!@!@
